packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -554,12 +554,7 @@ class DatePicker extends React.Component {
   }
 
   updateAriaLiveStatus(message) {
-    this.ariaLiveStatus = message;
```

Comment:
We can remove [this.ariaLiveStatus](https://github.com/cerner/terra-framework/blob/5a573cfa45752fe92e89859f8613343f46e4c1dd/packages/terra-date-picker/src/react-datepicker/index.jsx#L354) in the constructor since it is no longer needed.

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/866/commits/0c8cac409c12f490a2e487041b85970f376f2e39